Tribler/community/tunnel/hidden_community.py

```diff
@@ -107,10 +107,14 @@ def __init__(self, *args, **kwargs):
 
         self.intro_point_for = {}
         self.rendezvous_point_for = {}
+        self.infohash_rp_circuits = defaultdict(list)
+        self.seeding_introduction = {}
```

Comment:
This seeding_introduction should be a defaultdict(list), to be able to have more than one circuit for a single infohash